doc/en/api/1.0.0/gwcblobstores.yaml

```diff
@@ -0,0 +1,132 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Blobstores
+  description: BlobStores configure the persistance of tile data.
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /blobstores:
+    get:
+      operationId: blobstoresGet
+      summary: Get a list of configured blobstores
+      description: Displays a list of all configured blobstores on the server.
+      produces:
+        - application/xml
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/BlobStores"
+          examples:
+            application/xml: |
+              <blobStores>
+                <blobStore>
+                  <name>defaultCache</name>
+                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geowebcache/rest/blobstores/defaultCache.xml" type="text/xml"/>
+                </blobStore>
+              </blobStores>
+
+  /blobstores/{blobstoreName}:
+    get:
+      operationId: blobstoreGet
+      summary: Retrieve a configured blobstore
+      description: Retrieves a single configured blobstore definition.
+      produces:
+        - application/xml
+        - application/json
+      parameters:
+        - name: blobstoreName
+          in: path
+          required: true
+          description: The name of the blobstore to retrieve.
+          type: string
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/BlobStore"
+          examples:
+            application/xml: |
+              <FileBlobStore default="false">
+                <id>defaultCache</id>
+                <enabled>false</enabled>
+                <baseDirectory>/tmp/defaultCache</baseDirectory>
+                <fileSystemBlockSize>4096</fileSystemBlockSize>
+              </FileBlobStore>
+        404:
+          description: Unknown blobstore
+
+    put:
+      operationId: blobstorePut
+      summary: Create or update a configured blobstore.
+      description: Creates a new configured blobstore on the server, or modifies an existing blobstore.
+      parameters:
+        - name: blobstoreName
+          in: path
+          required: true
+          description: The name of the blobstore to add or update.
+          type: string
+        - name: blobstoreBody
+          in: body
+          description: The new blobstore definition.
+          required: true
+          schema:
+            $ref: "#/definitions/BlobStore"
+      consumes:
+        - application/xml
```

Comment:
In swagger editor it doesn’t dropdown the parameter content type application/xml in example, although declared in swagger doc. Might be a swagger limitation.

Reply:
Hm, odd. I can reproduce that issue in the swagger editor as well.

However, it seems to work fine on the hosted geoserver docs: http://docs.geoserver.org/latest/en/api/#/1.0.0/gwclayers.yaml

So I think it is just a problem with the editor.